services/webnn/ort/graph_builder_ort.cc

```diff
@@ -745,6 +748,98 @@ GraphBuilderOrt::AddInstanceNormalizationOperation(
   return base::ok();
 }
 
+[[nodiscard]] base::expected<void, mojom::ErrorPtr>
+GraphBuilderOrt::AddLayerNormalizationOperation(
+    const mojom::LayerNormalization& layer_normalization) {
+  const OperandDataType input_data_type =
+      GetOperand(layer_normalization.input_operand_id).descriptor.data_type();
+
+  const std::string input_name =
+      GetOperandName(layer_normalization.input_operand_id);
+  std::vector<const char*> input_names = {input_name.c_str()};
+
+  auto axes = layer_normalization.axes;
+  std::sort(axes.begin(), axes.end());
+  const auto axes_size = axes.size();
+  const std::vector<uint32_t>& input_shape =
+      GetOperand(layer_normalization.input_operand_id).descriptor.shape();
+  if (axes[axes_size - 1] != input_shape.size() - 1 ||
+      axes[0] != input_shape.size() - axes_size) {
```

Comment:
Please add a comment explaining it is good to only check beginning and ending of the ascending sorted axes, because the blink validation code ensures axes not having duplicated values.